__tests__/app.test.js

```diff
@@ -70,5 +72,37 @@ describe('/api/articles/:article_id', () => {
     .then(({body}) => {
       expect(body.msg).toBe('No article found under article_id 30')
     })
-  })  
+  })
 });
+describe('/api/articles', () => {
+  test('should respond with 200 status and return all articles without the body', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then((response) => {
+      expect(response.body).toHaveProperty('articles');
+      expect(Array.isArray(response.body.articles)).toBe(true);
```

Comment:
A length check would be really beneficial here. This way we can be sure that the forEach below won't give you a false positive